examples/00-mapdl-examples/modal_beam.py

```diff
@@ -0,0 +1,143 @@
+"""
+.. _ref_modal_beam:
+
+=================================
+MAPDL Modal Beam Analysis Example
```

Comment:
```suggestion
MAPDL modal beam analysis example
```
Sentence case titles per Google developer documentation style guide